orchestra/static/orchestra/task/js/autosave.js

```diff
@@ -0,0 +1,87 @@
+(function() {
```

Comment:
nitpick: this is a taskSaver with autosaving functionality.  so perhaps calling it a taskSaver and task_save.js might make more sense.

but that's just one person's opinion :)